NCrontab.Tests/CrontabScheduleTests.cs

```diff
@@ -54,12 +54,18 @@ public void CannotParseEmptyString()
         }
 
         [Test]
-        public void TryParseNullString() =>
+        public void TryParseNullString()
+		{
             Assert.That(CrontabSchedule.TryParse(null!), Is.Null);
+            Assert.False(CrontabSchedule.TryParse(string.Empty, out var _));
+        }
 
         [Test]
-        public void TryParseEmptyString() =>
+        public void TryParseEmptyString()
+        {
             Assert.That(CrontabSchedule.TryParse(string.Empty), Is.Null);
+            Assert.False(CrontabSchedule.TryParse(string.Empty, out var _));
```

Comment:
For consistency with the rest of the tests, use `Assert.That`:

```suggestion
            Assert.That(CrontabSchedule.TryParse(string.Empty, out var result), Is.False);
            Assert.That(result, Is.Null);
```

Also, checking that `result` is indeed `null` would be more thorough.